packages/admin/cms-admin/src/pages/pagesPage/PagesPage.tsx

```diff
@@ -80,6 +84,22 @@ export function PagesPage({
         stopPolling,
     });
 
+    if (error) {
+        const isPollingError = data !== undefined;
+
+        if (isPollingError) {
+            // Ignore
+        } else {
+            errorDialog?.showError({
```

Comment:
we could also throw error here

Reply:
This prevents us from showing a "beautiful" error message to the user. Or should we throw a custom error with a user message? Or throw the message itself?

Another way would be to define an error boundary around the page tree that shows the error message

Reply:
We said throwing error is usually the way to go.

An error boundary is needed of course, at least at page level. maybe also at stack/tab level?

Reply:
> We said throwing error is usually the way to go.

I now throw the error instead of showing an error dialog: 5f3046411e7b0c69696b4946c0d474571487efba.

> An error boundary is needed of course, at least at page level. maybe also at stack/tab level?

Added at the page level using `RouteWithErrorBoundary`. I've tried to add an additional ErrorBoundary to the pages page, but that doesn't catch the error. I will investigate this in an additional PR

Reply:
> I've tried to add an additional ErrorBoundary to the pages page, but that doesn't catch the error. I will investigate this in an additional PR

Nevermind, the error is thrown "above" the additional error boundary 😅